ember-server/src/main/scala/org/http4s/ember/server/EmberServerBuilder.scala

```diff
@@ -129,18 +135,51 @@ final class EmberServerBuilder[F[_]: Async] private (
     copy(requestHeaderReceiveTimeout = requestHeaderReceiveTimeout)
   def withLogger(l: Logger[F]) = copy(logger = l)
 
+  // If used will bind to UnixSocket
+  def withUnixSocketConfig(
+      unixSockets: UnixSockets[F],
+      unixSocketAddress: UnixSocketAddress,
+      deleteIfExists: Boolean = true,
+      deleteOnClose: Boolean = true) =
```

Comment:
Multiple parameters with defaults may hurt for binary compatibility in the future, but we can always give it a different name.  The alternative is to wrap this up in a type, but then _it_ needs a builder.  Nested, correlated options are nasty no matter what we do.